lib/fauna/objects.rb

```diff
@@ -128,4 +128,36 @@ def self.from_base64(enc)
       new(Base64.urlsafe_decode64(enc))
     end
   end
+
+  ##
+  # A QueryF.
+  #
+  # Reference: {FaunaDB Special Types}[https://fauna.com/documentation/queries-values-special_types]
+  class QueryF
```

Comment:
`F` feels confusing to me. In other drivers we've used `V` to express value types.